data.sql

```diff
@@ -14,3 +14,31 @@ VALUES
   ('Boarmon', '2005-06-07', 20.4, true, 7),
   ('Blossom', '1998-10-13', 17, true, 3),
   ('Ditto', '2022-05-14', 22, true, 4);
+
+  INSERT INTO owners (full_name, age) VALUES
+  ('Sam Smith', 34),
+  ('Jennifer Orwell', 19),
+  ('Bob', 45),
+  ('Melody Pond', 77),
+  ('Dean Winchester', 14),
+  ('Jodie Whittaker', 38);
+
+  INSERT INTO species (name) VALUES
+  ('Pokemon'),
+  ('Digimon');
+
+UPDATE animals SET species_id = 2 WHERE name LIKE '%mon';
+UPDATE animals SET species_id = 1  WHERE species_id IS NULL;
+
+UPDATE animals SET owner_id = 1 WHERE name='Agumon';
+UPDATE animals SET owner_id = 2 WHERE name='Gabumon';
+UPDATE animals SET owner_id = 2 WHERE name='Pikachu';
+UPDATE animals SET owner_id = 3 WHERE name='Devimon';
+UPDATE animals SET owner_id = 3 WHERE name='Plantmon';
+UPDATE animals SET owner_id = 4 WHERE name='Charmander';
+UPDATE animals SET owner_id = 4 WHERE name='Squirtle';
+UPDATE animals SET owner_id = 4 WHERE name='Blossom';
+UPDATE animals SET owner_id = 5 WHERE name='Angemon';
+UPDATE animals SET owner_id = 5 WHERE name='Boarmon';
```

Comment:
**OPTIONAL** - Good job with the work you have done so far, It is not advisable to hardcode the owner_id, what happens when you are working on a database that you didn't set up and you have to update the table?. Ideally, you should assign tables like this `SET owner_id = (SELECT id from owners WHERE full_name = 'Sam Smith')
   WHERE name = 'Agumon';`